cmd/faFilter/faFilter.go

```diff
@@ -63,6 +77,8 @@ func main() {
 	var name *string = flag.String("name", "", "Specifies the fasta record name.")
 	var notName *string = flag.String("notName", "", "Returns all fasta records except for this input.")
 	var minSize *int = flag.Int("minSize", 0, "Retains all fasta records with a sequence of at least that size")
+	var binFasta *int = flag.Int("binFasta", 0, "Makes specified number of bins from given fasta where all output files will contain similar amounts of sequence.")
+	var keepWhole *bool = flag.Bool("keepWhole", false, "Keep the output of faFilter before and after binning.")
```

Comment:
Can you help me understand what keepWhole does? Does it interact with the binFasta option?

Reply:
It looks like "keepWhole" flag being true is the same as "binFasta" = 0. I think this flag could be dropped. Unless there intended to be a role for keepWhole when binFasta != 0?

Reply:
thats exactly what it's for chelsea, I wanted to give an option to keep whatever filtered version of the genome was made before being broken into the bins, it's something I needed to keep the testing the way it was written, but also something I saw a use for down the line.